### DISCUSSION TITLE
[Feature] Overlay Label on Picture if specific Metadata Found

### DISCUSSION THREAD

**Discussion by TheCWB** [immich-app/immich#11679] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'd like to improve the "Remove Duplicate" utility. I'm proposing a pill shaped label be overlayed either on or below the photos, signifying whether certain metadata is found on the photo. The pill shape would have the name of the criteria in it, and either GREEN if the picture has it, and RED if it doesn't. 

Example: a GREEN "GPS" pill label overlayed / near a picture that contain GPS location data. Red if it does not. 

Some example criteria options: GPS, Camera Make / Model, Type (360 Degree Photo, Slo-Mo, etc.)

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by radh21301** [comment]
This might get messy when a user wants to have multiple metadata. Maybe easier if all this is listed below the picture. Right now, only the dimensions and the file size is shown. It also says if it is present in an album or not.

**Reply by TheCWB** [reply]
Yes, it could get messy if somebody wanted to enable a bunch, but my initial thought was that I only wanted GPS to show up, or HDR, etc. I could toggle on and off select labels to show during the determination process, to help determine what to keep. I initially had a program that stripped metadata from pictures, and now I’m going back to de-duping and only want to keep the ones with specific metadata still attached. 

Under the picture could potentially be a better place, so I’m not against the location if it shows it contains it.
